core/src/apps/common/keychain.py

```
@@ -101,7 +101,7 @@ def __del__(self) -> None:
         del self._cache
         del self.seed
 
-    def verify_path(self, path: paths.Bip32Path, force_strict: bool = False) -> None:
+    def verify_path(self, path: paths.Bip32Path, force_strict: bool = True) -> None:
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)